pkg/reconciler/taskrun/taskrun.go

```diff
@@ -185,7 +184,11 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, tr *v1beta1.TaskRun) pkg
 	// Reconcile this copy of the task run and then write back any status
 	// updates regardless of whether the reconciliation errored out.
 	if err = c.reconcile(ctx, tr, taskSpec, rtr); err != nil {
-		logger.Errorf("Reconcile error: %v", err.Error())
+		if merr, ok := err.(*multierror.Error); ok {
```

Comment:
I think you can just print the `merr.Error()` and it will print them all in a list for you?  Is that not happening?

Reply:
That works!